core/model/src/driver.rs

```diff
@@ -236,3 +236,92 @@ impl RpcMessage for ValidateAllocation {
     type Item = bool;
     type Error = GenericError;
 }
+
+// ************************** ENTER **************************
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Enter {
+    amount: String,
+    network: Option<String>,
+    token: Option<String>,
+}
+
+impl Enter {
+    pub fn new(amount: String, network: Option<String>, token: Option<String>) -> Enter {
+        Enter {
+            amount,
+            network,
+            token,
+        }
+    }
+}
+
+impl RpcMessage for Enter {
+    const ID: &'static str = "Enter";
+    type Item = String; // Transaction Identifier
+    type Error = GenericError;
+}
+
+// ************************** EXIT **************************
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Exit {
+    to: Option<String>,
+    amount: Option<String>,
+    network: Option<String>,
+    token: Option<String>,
+}
+
+impl Exit {
+    pub fn new(
+        to: Option<String>,
+        amount: Option<String>,
```

Comment:
Parsing amount should be done in CLI, not in driver.
```suggestion
        amount: Option<BigDecimal>,
```